src/main/java/ch/uzh/ifi/fs22/sel/group15/cozycave/server/service/ApplicationService.java

```diff
@@ -0,0 +1,182 @@
+package ch.uzh.ifi.fs22.sel.group15.cozycave.server.service;
+
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.constant.ApplicationStatus;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.constant.Role;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.entity.applications.Application;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.entity.listings.Listing;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.entity.users.User;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.repository.ApplicationRepository;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.repository.ListingRepository;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.repository.UserRepository;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service @Transactional public class ApplicationService {
+
+    private final Logger log = LoggerFactory.getLogger(ApplicationService.class);
```

Comment:
you can use Lombok instead

Reply:
have you any links for that?

Reply:
https://projectlombok.org/features/log thus @Slf4j